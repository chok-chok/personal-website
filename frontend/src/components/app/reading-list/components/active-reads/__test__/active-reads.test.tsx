import * as React from 'react';
import ActiveReads from '../active-reads';
import { shallow } from 'enzyme';

describe('Test ActiveReads component', () => {
  it('should renders correctly', () => {
    const tree = shallow(<ActiveReads />);

    expect(tree).toMatchSnapshot();
  });
});
