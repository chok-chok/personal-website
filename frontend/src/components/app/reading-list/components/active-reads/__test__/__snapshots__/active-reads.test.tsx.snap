// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ActiveReads component should renders correctly 1`] = `
<Fragment>
  <styled__StyledH2>
    Currently reading
  </styled__StyledH2>
  <styled__ActiveBookWrapper>
    <styled__BookImg
      src="/img/example-book-img.jpg"
    />
    <styled__BookDetailWrapper>
      <styled__BookLink
        href="https://www.amazon.com"
        target="_blank"
      >
        <styled__BookTitle>
          Domain Driven Design: Tackling Complexity in the Heart of Software
        </styled__BookTitle>
      </styled__BookLink>
      <styled__AuthorInfo>
        by Eric Evans
      </styled__AuthorInfo>
    </styled__BookDetailWrapper>
  </styled__ActiveBookWrapper>
</Fragment>
`;
